Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs

```diff
@@ -2,14 +2,24 @@
 // Licensed under the MIT License. See LICENSE in the project root for
 // license information. 
 
+using Microsoft.Extensions.PlatformAbstractions;
+using System;
+using System.Reflection;
+
 namespace Artesian.SDK.Service
 {
     /// <summary>
     /// Artesian Constants
     /// </summary>
     public abstract class ArtesianConstants
     {
-        internal const string CharacterValidatorRegEx = @"^[^'"",:;\s](?:(?:[^'"",:;\s]| )*[^'"",:;\s])?$";
+        internal const string SDKVersion = "v2.2.1";
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+		public static string SDKVersionHeaderValue = $@".NET<{Assembly.GetExecutingAssembly().GetName().Version}>,{Environment.OSVersion.Platform}<{Environment.OSVersion.Version}>,{PlatformServices.Default.Application.RuntimeFramework.Identifier}<{PlatformServices.Default.Application.RuntimeFramework.Version}>";
```

Comment:
The original code had also a 'v' in front of the version number. Not sure that is the expected convention. Ask @sicurezza if is v1.2.3 or just 1.2.3. 

If unspecified, please specify it and make sure existing implementations uses the chosen format.